packages/cli/tests/unit/inspect.spec.ts

```diff
@@ -11,9 +11,9 @@ import { CodeObject, CodeObjectMatch } from '../../src/search/types';
 tmp.setGracefulCleanup();
 
 const fixtureDir = join(__dirname, 'fixtures', 'ruby');
-const appMapDir = tmp.dirSync().name;
+const appMapDir = tmp.dirSync().name.split('\\').join('/');
```

Comment:
Or `replaceAll` ;)

Reply:
It is weird. Too old a typescript or some type definitions? But I'd think they'd be pinned by yarn.